tests/root_gauge_factory_proxy/test_root_gauge_factory_proxy_admin.py

```diff
@@ -0,0 +1,275 @@
+import brownie
+from brownie import ETH_ADDRESS
+
+
+def test_set_manager(root_gauge_factory_proxy, alice, bob, charlie, chain):
+
+    default_ownership_admin = root_gauge_factory_proxy.ownership_admin()
+    default_emergency_admin = root_gauge_factory_proxy.emergency_admin()
+    assert root_gauge_factory_proxy.manager() == alice  # contract deployer is manager
+
+    # unauthorised users cannot change the manager:
+    with brownie.reverts():
+        root_gauge_factory_proxy.set_manager(charlie, {"from": bob})
+
+    # emergency admin and ownership admin can become manager:
+    for acct in [default_emergency_admin, default_ownership_admin]:
+        root_gauge_factory_proxy.set_manager(acct, {"from": acct})
+        assert root_gauge_factory_proxy.manager() == acct
+        chain.undo()
+
+    # authorised users can change the manager to anyone:
+    for acct in [default_emergency_admin, default_ownership_admin, alice]:
+        root_gauge_factory_proxy.set_manager(bob, {"from": acct})
+        assert root_gauge_factory_proxy.manager() == bob
+        chain.undo()
+
+
+def test_transfer_proxy_admins(root_gauge_factory_proxy, alice, bob, charlie):
```

Comment:
This test is loaded lol, should break up into appropriate parts.
- Revert unauthorized commit
- Success authorized commit
- Revert unauthorized accept
- Success authorized accept